client/src/components/Landing/Landing.js

```diff
@@ -33,9 +34,16 @@ const Landing = () => {
       setIsLoading(true);
       setShowDescription(false);
 
-      // const TRANSLATE_END_POINT = "";
+      const ENDPOINT = `?sl=${formData.sl}&dl=${formData.dl}&code=${formData.code}`;
 
-      history.push("?sl=c&dl=javascript&code=console.log()");
+      const results = await fetch(`http://localhost:5000/api/v1${ENDPOINT}`);
+      const data = await results.json();
+
+      if (data) {
+        setTranslations(data);
+      }
+
+      history.push(ENDPOINT);
```

Comment:
later I want to see if I can get the new url to just be "localhost:PORT?sl=..." instead of what it is now 👇 
![image](https://user-images.githubusercontent.com/53862535/117512317-610d1a00-af5d-11eb-9176-bdc8ef5b1bad.png)